python/ray/state.py

```diff
@@ -866,6 +866,15 @@ def nodes():
     return state.node_table()
 
 
+def workers():
```

Comment:
I think this needs to be added here;
https://github.com/ray-project/ray/blob/0fbee4da0c9bdc57a40d424d7dde2213991e1744/python/ray/__init__.py#L107

Reply:
updated.